### DISCUSSION TITLE
Position of tooltips is a bit off

### DISCUSSION THREAD

**Discussion by prologic** [picocss/pico#163] [Help]
Hey all 👋

Just want some ideas. / opinions on the position of tooltip. We use Pico CSS as the base CSS library for Yarn.social -- We noticed lately that tooltips get a bit cut-off or are poorly positions. See screenshot.

Example link: https://twtxt.net/twt/miirjbq

Is this something we can address in Pico?

<img width="1392" alt="Screen Shot 2022-02-20 at 08 31 33" src="https://user-images.githubusercontent.com/1290234/154821259-3460f32d-f3ec-43a0-8497-5111db4bd73a.png">

**Comment by lucaslarroche** [comment]
@prologic,
This issue will be fixed in the [next release](https://github.com/picocss/pico/issues/155)
#156 #155
